slices/less.yaml

```diff
@@ -0,0 +1,32 @@
+package: less
+
+essential:
+  - less_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/less:
+      /usr/bin/lessecho:
+      /usr/bin/lesskey:
+
+  mime:
```

Comment:
I think we can omit adding a slice for this.
The contents os `/usr/lib/mime/packages/less` doesn't look very compelling.